### ISSUE TITLE
yade.sh overwrites file in $JS7_RETURN_VALUES and does not append \n to own property

### ISSUE THREAD

**Issue by rfx77** [sos-berlin/js7#8]
1) when you execute yade.sh the file in $JS7_RETURN_VALUES ist completely overwritten with the contents of the yade return value. 
so when you add custom return values to the file before they are gone

2) yade.sh does not add a linebreak to its own variable so you cannot simple add your own afterwards with the method in the documentation. you have to add a linebreak first.

the usecase for custom return values is to get a a filelist out of a job.

here is my workaround:

```
function yade_getlist() {

  TEMP=$(mktemp)
  RET=$(mktemp)

  $JS7_YADE_BIN \
    -settings $1 \
    -profile $2 \
    -return-values=$RET \
    -result_set_filename=$TEMP

  echo $(cat $RET) >> $JS7_RETURN_VALUES
  echo "filelist=$(cat $TEMP |base64)" >> $JS7_RETURN_VALUES

  rm -f $RET
  rm -f $TEMP

}

```

**Comment by sosap** [comment]
This works as designed. For now in a YADE job the only accepted return value is specific for YADE and is added by JOC Cockpit to the File Transfer view.

We understand that users want to forward variables to subsequent jobs, for example the path to a file holding results of a GETLIST operation. This is considered an improvement that will be made available from the following change:

* [YADE-606: Allow additional return values for YADE jobs](https://change.sos-berlin.com/browse/YADE-606)

The improvement will become available with [Release 1.13.20](https://kb.sos-berlin.com/x/04jqBg)

**Comment by sosap** [comment]
Just for the records: when forwarding a variable (holding a list of files) to subsequent jobs then there are two reasons why to base64 encode the value of the variable:

* to prevent included special characters such as $, \n, \r etc. to break use with subsequent shell jobs that read the value from an OS environment variable.
* to allow a list of arbitrary length to be used:
    * if the subsequent job is derived from [JS7 - JITL Job Templates](https://kb.sos-berlin.com/display/JS7/JS7+-+JITL+Job+Templates) then this allows to use arbitrary length values
    * if the subsequent job is a shell job then the value should not be base64 encoded as this doubles the value's size while OS environment variables as used by the receiving job are limited in size depending on the OS in use. It might be preferable to write the list to a file and to forward its path to subsequent jobs.
* find details from the docs: [JS7 - How to pass variables with arbitrary length values to jobs](https://kb.sos-berlin.com/display/JS7/JS7+-+How+to+pass+variables+with+arbitrary+length+values+to+jobs)

For use with subsequent JITL jobs modify the following line in the above example

`echo "filelist=$(cat $TEMP |base64)" >> $JS7_RETURN_VALUES`

to

`echo "filelist=base64:$(< $TEMP base64)" >> $JS7_RETURN_VALUES`
